### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mrteey**
- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

Description:

It's super annoying to copy a command and have to remove $ sign before you run it, this update removes the $ sign from the README.md file to allow users to easily copy and run commands.

### CODE REVIEW

**Review by picklelo — APPROVED** [review]
I think we had that originally so that it's clear these are bash commands, but I think it will still be clear

**Comment by mrteey** [comment]
> I think we had that originally so that it's clear these are bash commands, but I think it will still be clear

It will be, most people coming to GitHub already know about these things.

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Pynecone requires the following to get started:
 -   [Node.js 16.6.0+](https://nodejs.org/en/) (Don't worry, you won’t have to write any JavaScript!)
 
 ```
```
Comment (masenf) [inline README.md:24]: Might add `sh` after the backticks as a hint to readers and the highlighter that this is shell snippet.

**Review by masenf — APPROVED** [review]
Much easier to copy paste without the dollar signs

**Pull request merged**